### PR TITLE
🐛 Fix notificationCount not including mint_nft

### PR DESCRIPTION
**Pull request by AuroraHuang22**

### CODE REVIEW

File: src/store/modules/wallet.js [hunk 1]
```diff
@@ -157,14 +157,21 @@ const getters = {
...
-    return getters.getEvents.filter(
-      e =>
+    return getEvents.reduce((count, e) => {
+      if (
         state.eventLastSeenTs < new Date(e.timestamp).getTime() &&
-        (e.eventType === 'nft_sale' || e.eventType === 'receive_nft')
-    ).length;
+        (e.eventType === 'nft_sale' ||
+          e.eventType === 'receive_nft' ||
+          e.eventType === 'mint_nft')
```
Comment (nwingt) [inline src/store/modules/wallet.js:169]: Use `Array.prototype.includes`?

**Review by nwingt — APPROVED** [review]

File: src/store/modules/wallet.js [hunk 2]
```diff
@@ -157,14 +157,19 @@ const getters = {
...
       return 0;
     }
-    return getters.getEvents.filter(
-      e =>
+    return getEvents.reduce((count, e) => {
+      if (
         state.eventLastSeenTs < new Date(e.timestamp).getTime() &&
-        (e.eventType === 'nft_sale' || e.eventType === 'receive_nft')
-    ).length;
+        ['nft_sale', 'receive_nft', 'mint_nft'].includes(e.eventType)
```
Comment (williamchong) [inline src/store/modules/wallet.js:167]: Aren't we using new_class event though

**Review by williamchong — APPROVED** [review]

File: src/store/modules/wallet.js (hunk 2 above)
Comment (AuroraHuang22) [reply src/store/modules/wallet.js:167]: have formatted [eventType](https://github.com/likecoin/liker-land/blob/develop/src/store/modules/wallet.js#L180-L181)

Comment (nwingt) [reply src/store/modules/wallet.js:167]: Maybe the event type should be declared as constant for reuse

**Pull request merged**